janitor/config/defaults.py

```diff
@@ -7,7 +7,6 @@
 LOCALHOST = os.environ.get("LOCALHOST", "127.0.0.1")
 
 SYNC_JOB_INTERVAL_SEC = 300
-SYNC_JOB_OVERLAP_SEC = 10
```

Comment:
So initially, we were running the job every 5 minutes and would retrieve all updated rows in the last 5 minutes (i.e. since the job ran the last time), and there was a remote possibility that one might update an entry at the exact moment the job runs and therefore may not be picked up. That is why we added the overlap, because even if it checks the same entries more than once, it would only update the `updated_at` column and would have no impact on anything else.

However, now we are getting all the updates since the last updated entry in the new table, so no interval or overlap is required in the query. So if `janitor` was offline for half an hour for instance, running it would pick up all updates that occurred during that time without having to specify a time window. The job interval is still required to set how often the job itself runs, but the overlap was only used in the SQL query which now uses neither.